env_example.txt

```diff
@@ -1,13 +1,14 @@
-# Environment variables go here, can be read by `python-dotenv` package:
-#
-#   `src/script.py`
-#   ----------------------------------------------------------------
-#    import dotenv
-#
-#    project_dir = os.path.join(os.path.dirname(__file__), os.pardir)
-#    dotenv_path = os.path.join(project_dir, '.env')
-#    dotenv.load_dotenv(dotenv_path)
-#   ----------------------------------------------------------------
-#
-# DO NOT ADD THIS FILE TO VERSION CONTROL!
-CAT_INDIVIDUALS_DS_PATH='!!!!WRITE PATH FOR INDIVID CATS DATASETS!!!!!'
\ No newline at end of file
+PROJECT_DIR=''
+#for train
+CAT_INDIVIDUALS_DS_PATH=''
+OUTPUT_FILEPATH=''
+
+# telegram bot
+BOT_TOKEN=''
+
```

Comment:
Could you please descriptions for these variables and where we can grab value of them